### ISSUE TITLE
No stdin in Cargo output window. 

### ISSUE THREAD

**Issue by Emilgardis** [saviorisdead/RustyCode#88]
Using the first [example][link] in the book of the guessing game (and therefore anything using stdin), the cargo output window does not accept any input.

==Steps to recreate==
1. `cargo new guessing_game --bin`
2. open folder in vscode and paste the [code][link] into main.rs
3. Run

[link]: http://doc.rust-lang.org/book/guessing-game.html#processing-a-guess

**Comment by KalitaAlexey** [comment]
@Emilgardis VSCode does not support input into `OutputChannel`.

**Comment by Emilgardis** [comment]
Fair enough, however, I'm fairly certain that there is another channel for output that supports it. Is there any interest to include that?

**Comment by KalitaAlexey** [comment]
@Emilgardis Yes. Sometimes I miss that.

**Comment by DerGroncki** [comment]
Yes, would be great.

**Comment by JasonBock** [comment]
So far I'm resorting to using a terminal window to navigate to the exe and running it there. Cumbersome, but it works. Would be nice if the Run command did this.
